examples/tictactoe/src/components/proof-queue.tsx

```diff
@@ -0,0 +1,37 @@
+import { type FC } from "react";
+import { LoadingSpinner } from "./loading-spinner";
+
+type ProofQueueProps = {
+  assertionQueue: string[];
+};
+
+export const ProofQueue: FC<ProofQueueProps> = ({ assertionQueue }) => {
+  return (
+    <div>
+      <h3>Proof Queue</h3>
+      {assertionQueue.length === 0 && <p>No proofs are being generated</p>}
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "center",
+          alignItems: "center",
+          gap: "1rem",
+        }}
+      >
+        {assertionQueue.length === 1 && (
+          <>
+            <LoadingSpinner isSmall />
+            <p> {assertionQueue.length} proof still needs to be generated</p>
+          </>
+        )}
+        {assertionQueue.length > 1 && (
+          <>
+            <LoadingSpinner isSmall />
+            <p> {assertionQueue.length} proofs still needs to be generated</p>
```

Comment:
Keep it DRY. The only change is on "proof(s)". So better parametrise just that word, instead of duplicating the spinner and whole paragraph.